projects/pastanaga/src/lib/textfield/textfield.common.ts

```diff
@@ -16,22 +16,22 @@ export class TextfieldCommon implements ControlValueAccessor, OnInit, Validator
     @Input() placeholder?: string;
     @Input() help?: string;
     @Input()
-    get required(): boolean { return this._required; }
-    set required(value: boolean) { this._required = coerceBooleanProperty(value); }
+    get isRequired(): boolean { return this._required; }
```

Comment:
we want to keep `required` and `disabled` as input name. So you can't rename the getter/setter. you can rename the protected property itself if you need though.

Reply:
the TextfieldCommon on front-ui has isRequired, should i change it on there?

Reply:
no, the whole point is to use pastanaga instead of front-ui. When we will replace our `o-textarea` by `pa-textarea` we will just need to rename inputs accordingly.
At the end textarea and inputs will be removed from front-ui